dask/tests/test_threaded.py

```diff
@@ -105,58 +103,28 @@ def test_f():
     assert L == [1] * 20
 
 
-code = """
-import sys
-import time
-from dask.threaded import get
-
-def signal_started():
-    sys.stdout.write('started\\n')
-    sys.stdout.flush()
-
-def long_task(x):
-    time.sleep(5)
-
-dsk = {('x', i): (long_task, 'started') for i in range(100)}
-dsk['started'] = (signal_started,)
-dsk['x'] = (sum, list(dsk.keys()))
-get(dsk, 'x')
-"""
-
-
 # TODO: this test passes locally on windows, but fails on appveyor
 # because the ctrl-c event also tears down their infrastructure.
 # There's probably a better way to test this, but for now we'll mark
 # it slow (slow tests are skipped on appveyor).
 @pytest.mark.slow
```

Comment:
Could be wrong, but it looks like it [worked]( https://ci.appveyor.com/project/daskdev/dask/build/1.0.1133#L2832 ). As it was using this [merge commit]( https://github.com/dask/dask/commit/14c69704ef2fc2121bf8518ae8678143061c5c2c ) to build, we know it [shouldn't be skipping]( https://github.com/dask/dask/commit/14c69704ef2fc2121bf8518ae8678143061c5c2c#diff-b747a54283b27c9c5bb73e56aa2fe556L131 ) it. Though am not totally sure as we still see one skip in that test.